packages/components/organisms/f-checkout/src/assets/images/jet-sad-bag.svg

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please can you run this through [SVGOMG](https://jakearchibald.github.io/svgomg/)?
Using `precision = 2` cuts the size by almost 70% without any appreciable drop in quality.